src/jazzlights/effect/creatures.cpp

```diff
@@ -127,17 +147,21 @@ void Creatures::begin(const Frame& frame) const {
 
 // Called once for each point in time to prepare the state before calling color() for each pixel.
 void Creatures::rewind(const Frame& frame) const {
-  const Milliseconds currentTime = timeMillis();
+  KnownCreatures::Get()->update();
   const std::vector<Creature>& creatures = KnownCreatures::Get()->creatures();
   size_t num_creatures = creatures.size();
   if (num_creatures > kMaxNumColors - 2) { num_creatures = kMaxNumColors - 2; }
   uint32_t num_close_creatures = 0;
   for (size_t i = 0; i < num_creatures; i++) {
     // Compute the color for each known creature.
     const Creature& creature = creatures[i];
-    uint8_t intensity = CreatureIntensity(creature, currentTime);
+    uint8_t intensity = CreatureIntensity(creature);
     state(frame)->colors[i] = FadeColor(CRGB(creature.color), intensity);
-    if (intensity >= 100) { num_close_creatures++; }
+    if (creature.lastHeardNearby < 0 ||
+        timeMillis() - creature.lastHeardNearby < KnownCreatures::kNearbyCreatureTimeoutMs) {
+      // If the creature has been heard nearby in the last KnownCreatures::kNearbyCreatureTimeoutMs it is still close
```

Comment:
nit: add period to comments
```suggestion
      // If the creature has been heard nearby in the last KnownCreatures::kNearbyCreatureTimeoutMs it is still close.
```